Xamarin.Essentials/Permissions/Permissions.ios.tvos.watchos.cs

```diff
@@ -125,9 +125,11 @@ internal static PermissionStatus GetLocationStatus(bool whenInUse)
                 };
             }
 
+            static CLLocationManager locationManager;
+
             internal static Task<PermissionStatus> RequestLocationAsync(bool whenInUse, Action<CLLocationManager> invokeRequest)
             {
-                var locationManager = new CLLocationManager();
```

Comment:
What happens if we come back in here without waiting for the flow to complete? We might leak...